DocGen4/Output/Module.lean

```diff
@@ -199,6 +199,9 @@ def moduleToHtml (module : Process.Module) : HtmlM Html := withTheReader SiteBas
   let relevantMembers := module.members.filter Process.ModuleMember.shouldRender
   let memberDocs ← relevantMembers.mapM (moduleMemberToHtml module.name)
   let memberNames := filterDocInfo relevantMembers |>.map DocInfo.getName
+  letI : MonadLift BaseHtmlM HtmlM := {
```

Comment:
Is this needed? I would have expected Lean to infer this instance on its own.

Reply:
I checked again and unfortunately it's still needed, as we already discussed https://github.com/leanprover/doc-gen4/pull/200#discussion_r1650098285

Reply:
Okay I know how to fix it, will merge this then fix it myself.